python/tests/test_gemm.py

```diff
@@ -277,8 +284,12 @@ def matmul_kernel(
                         stride_am=a.stride(0), stride_ak=a.stride(1),
                         stride_bk=b.stride(0), stride_bn=b.stride(1),
                         stride_cm=c.stride(0), stride_cn=c.stride(1),
-                        BLOCK_SIZE_M=block_M, BLOCK_SIZE_N=block_N, BLOCK_SIZE_K=block_K)
+                        BLOCK_SIZE_M=block_M, BLOCK_SIZE_N=block_N, BLOCK_SIZE_K=block_K, ALLOW_TF32=allow_tf32)
 
     golden = torch.matmul(a, b)
     golden_abs_err, golden_rel_err = get_proper_err(a, b, golden)
-    torch.testing.assert_close(c, golden, rtol=max(1e-4, 1.5 * golden_rel_err), atol=max(1e-4, 1.5 * golden_abs_err))
```

Comment:
https://github.com/openai/triton/blob/8650b4d1cbc750d659156e2c17a058736614827b/python/test/unit/language/test_core.py#L1148

The legacy code uses a large atol.